src/compas_rhino/install_with_pip.py

```diff
@@ -1,20 +1,117 @@
+import pathlib
+import random
+import string
 import subprocess
 
-import compas_rhino
+import click
 
+rhinocode = pathlib.Path().home() / ".rhinocode"
+rhinopython = rhinocode / "py39-rh8" / "python3.9"
+site_envs = rhinocode / "py39-rh8" / "site-envs"
```

Comment:
I personally find pathlib syntax abuse annoying AF but that's very particular of my own quirks, feel free to ignore this one 😂